### ISSUE TITLE
Black screen with Maui Blazor Hybrid in Net 7

### ISSUE THREAD

**Issue by ChristopherRecinos** [dotnet/maui#13291]
### Description

Getting consistent black screens when debugging using .Net Maui Blazor Hybrid with .Net 7. 

Have to restart the build a few times before it fixes itself and renders properly. This is happening with a completely new project.

### Steps to Reproduce

1. Create a brand new project for .Net Maui Blazor App with .net 7
2. Hit F5 or Play

![Capture](https://user-images.githubusercontent.com/77091288/218388438-ca036136-3b1e-4e93-b910-41ff9ab6ee57.JPG)

### Link to public reproduction project repository

https://ChristopherRecinos@dev.azure.com/ChristopherRecinos/MauiBlazorBlackScreen/_git/MauiBlazorBlackScreen

### Version with bug

7.0 (current)

### Last version that worked well

6.0.312

### Affected platforms

Windows

### Affected platform versions

Windows SDK 10.0.19041

### Did you find any workaround?

I have not found any workaround to this other than restarting the app several times.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us. Do you see this happening with non-Blazor app - just MAUI?

**Comment by ChristopherRecinos** [comment]
@mkArtakMSFT I just made a brand new Maui app, non-blazor, and it does it as well. I even tried the projects on another computer and get the same result.

![image](https://user-images.githubusercontent.com/77091288/223197088-23429acd-802d-4c44-a664-a0c680137ad4.png)

**Comment by danroth27** [comment]
@Redth Is this a known issue?

**Comment by mattleibow** [comment]
I have not seen this before. Is this on a machine with the monitor connected or via some RDP/remote/streaming display?

Can you reproduce this on other machines or with a plain WinUI 3 app?

**Comment by ChristopherRecinos** [comment]
@mattleibow  This is on just my regular everyday computer. I've made sure no background apps or anything are running that could affect it. I also tried it on my second machine and it happened there too, however it did seem to not happen on my work computer in the office.  I just don't know what the issue could be. I could try reinstalling all of visual studios potentially?

I have not tried a plain WinUI 3 app but I will try that today.
